lib/libutee/arch/arm/utee_mcount_a32.S

```diff
@@ -52,3 +52,15 @@ FUNC __utee_mcount, :
 END_FUNC __utee_mcount
 
 #endif /* CFG_TA_GPROF_SUPPORT */
+
+#ifdef CFG_TA_FTRACE_SUPPORT
+/*
+ * ToDo: ftrace support to be added for 32 bit apps. Currently this is just a
```

Comment:
s/ToDo/TODO/

I think this TODO should be replace by a simple statement: `ftrace is not yet supported in 32bit mode`

Reply:
Ok. BTW, I am planning to work on it once we get this patch-set reviewed.